lib/alephant/broker/response/batch.rb

```diff
@@ -35,7 +35,9 @@ def json
               "status"       => component.status,
               "content_type" => component.content_type,
               "body"         => component.content
-            }
+            }.tap do |headers|
+              headers["seq_id"] = component.headers["X-Sequence"] if component.headers["X-Sequence"]
```

Comment:
Just being a pedant, but could we use `sequence_id` or is there already something expecting `seq_id`?